common/djangoapps/third_party_auth/tests/specs/test_generic.py

```diff
@@ -7,7 +7,7 @@
 from .base import IntegrationTestMixin
 
 
-@unittest.skipUnless(testutil.AUTH_FEATURE_ENABLED, 'third_party_auth not enabled')
+@unittest.skipUnless(testutil.AUTH_FEATURE_ENABLED, testutil.AUTH_FEATURES_KEY + ' not enabled')
```

Comment:
@bradenmacdonald Oo, I agree. Let's keep it for another PR? I'm refactor-savvy so I'd do it outside of any particular ticket.